backend/helper_zero/models.py

```diff
@@ -1,5 +1,16 @@
 from django.db import models
 
+class User(models.Model):
+  name = models.CharField(max_length=120, null=True)
```

Comment:
get rid of all the null=True if blank is not set

Reply:
Resolved in person